DbTables/inc/DbSet.hh

```diff
@@ -0,0 +1,60 @@
+#ifndef DbTables_DbSet_hh
+#define DbTables_DbSet_hh
+
+// A calibration set as it is stored and maintained in the 
+// database includes pointers from a purpose, version, extension,
+// and groups down to the list of IoVs.  This container 
+// holds just the list of IoVs which is the result of flattening the 
+// heirarchical structure.  For faster and simpler lookup and manipulation
+
+
+#include "Offline/DbTables/inc/DbIoV.hh"
+#include <map>
+#include <vector>
+
+
+namespace mu2e {
+
+  class DbSet {
+  public:
+
+    // IoV extended with its related cid
+    class EIoV {
+    public:
+      EIoV():_cid(-1) {}
+      EIoV(int cid, DbIoV const& iov):_cid(cid),_iov(iov) {}
+      int cid() const {return _cid;}
+      DbIoV const& iov() const {return _iov;}
+      void setCid(int cid) {_cid = cid;}
+      void setIov(DbIoV const& iov) {_iov = iov;}
```

Comment:
Are there good use cases for the two setter functions?  Unless there are good use, cases I prefer to leave them out in order to force single phase construction.

Reply:
one deprecated use case, I'll remove them